tests/acceptance/browzine-summon-adapter-test.js

```diff
@@ -383,6 +383,87 @@ describe("BrowZine Summon Adapter >", function() {
       });
     });
 
+    describe("search results article with an article in context link but no direct to pdf link and no article link >", function() {
+      beforeEach(function() {
+        browzine.articleLinkEnabled = true;
+        summon = browzine.summon;
+
+        documentSummary = $("<div class='documentSummary' document-summary><div class='coverImage'><img src=''/></div><div class='docFooter'><div class='row'></div></div></div>");
+
+        inject(function ($compile, $rootScope) {
+          $scope = $rootScope.$new();
+
+          $scope.document = {
+            content_type: "Journal Article",
+            dois: ["10.1136/bmj.h2575"]
+          };
+
+          documentSummary = $compile(documentSummary)($scope);
+        });
+
+        jasmine.Ajax.install();
+
+        summon.adapter(documentSummary);
+
+        var request = jasmine.Ajax.requests.mostRecent();
+
+        request.respondWith({
+          status: 200,
+          contentType: "application/json",
+          response: JSON.stringify({
+            "data": {
+              "id": 55134408,
+              "type": "articles",
+              "title": "New England Journal of Medicine reconsiders relationship with industry",
+              "date": "2015-05-12",
+              "authors": "McCarthy, M.",
+              "inPress": false,
+              "availableThroughBrowzine": true,
+              "startPage": "h2575",
+              "endPage": "h2575",
+              "browzineWebLink": "https://browzine.com/libraries/XXX/journals/18126/issues/7764583?showArticleInContext=doi:10.1136/bmj.h2575",
+              "fullTextFile": "",
+              "contentLocation": ""
+            },
+            "included": [{
+              "id": 18126,
+              "type": "journals",
+              "title": "theBMJ",
+              "issn": "09598138",
+              "sjrValue": 2.567,
+              "coverImageUrl": "https://assets.thirdiron.com/images/covers/0959-8138.png",
+              "browzineEnabled": true,
+              "browzineWebLink": "https://develop.browzine.com/libraries/XXX/journals/18126"
+            }]
+          })
+        });
+
+        expect(request.url).toMatch(/articles\/doi\/10.1136%2Fbmj.h2575/);
+        expect(request.method).toBe('GET');
+      });
+
+      afterEach(function() {
+        jasmine.Ajax.uninstall();
+        delete browzine.articleLinkEnabled;
+      });
+
+      it("should not enhance the search result with an article in context link", function() {
+        var template = documentSummary.find(".browzine");
+
+        expect(template).toBeDefined();
+
+        expect(template.text().trim()).not.toContain("Article PDF: Download Now");
```

Comment:
Instead of `Download Now`, would this text probably say `Download PDF` by default?

I don't see any test fixture setup where we're using non-default text here... think we should update this text to reflect the new default?

```suggestion
        expect(template.text().trim()).not.toContain("Article PDF: Download PDF");
```

Reply:
> Instead of Download Now, would this text probably say Download PDF by default?

Summon's default text is different from Primo's default text.

Summon: `Article PDF: Download Now`
Primo: `Download PDF`